src/components/modules/banners/globalEoy/components/GlobalEoyBody.tsx

```diff
@@ -31,10 +31,10 @@ const GlobalEoyBody: React.FC<GlobalEoyBodyProps> = ({
                     <Hide below="tablet">
                         {shouldShowArticleCount ? (
                             <>
-                                In an extraordinary 2020, our independent journalism was powered by
-                                more than a million supporters. Thanks to you, we provided vital
-                                news and analysis for everyone, led by science and truth.
-                                You&apos;ve read{' '}
+                                In the extraordinary year that was 2020, our independent journalism
```

Comment:
the linter has messed up the diff here, all that's changed is:
"In an extraordinary 2020"
to
"In the extraordinary year that was 2020"